FrostyEditor/FrostyEditor.csproj

```diff
@@ -7,6 +7,20 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="Assets\FolderCollapsed.png" />
+    <None Remove="Assets\FolderExpanded.png" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ApplicationDefinition Include="Assets\FolderCollapsed.png" />
```

Comment:
any reason why this is also an applicationdefinition?

Reply:
not sure, it does that when i set the resource type to AvaloniaResource

Reply:
weird